vernac/comTactic.mli

```diff
@@ -0,0 +1,47 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *         Copyright INRIA, CNRS and contributors             *)
+(* <O___,, * (see version control and CREDITS file for authors & dates) *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+
+(** Tactic interpreters have to register their interpretation function *)
+type 'a tactic_interpreter
+type interpretable = I : 'a tactic_interpreter * 'a -> interpretable
+
+(** ['a] should be marshallable if ever used with [par:] *)
+val register_tactic_interpreter :
+  string -> ('a -> unit Proofview.tactic) -> 'a tactic_interpreter
+
```

Comment:
Who are users of this registration function? `mtac` , `coq-elpi` ?

Reply:
Apart from ltac, that is.

Reply:
Ideally yes, but as of today I believe that the "Extensible AST" is ltac1 specific (I'd be happy to br wrong).
If we change that, then other tactic languages could plug their AST and interpreters.

Reply:
(I mean, at the parser level, not in this API)

Reply:
Yeah, I guess that will become clear if the isolation of tactics does work, so indeed there is not need to call solve from plugin code.